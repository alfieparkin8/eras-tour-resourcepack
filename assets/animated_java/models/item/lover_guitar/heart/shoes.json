{"parent":"animated_java:item/lover_guitar/shoes","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}