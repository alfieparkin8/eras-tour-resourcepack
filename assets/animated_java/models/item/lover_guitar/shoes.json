{
	"textures": {
		"5": "animated_java:item/lover_guitar/sunset_boots_layer_1"
	},
	"elements": [
		{
			"from": [
				6.65,
				7.45,
				6.25
			],
			"to": [
				12.15,
				20.95,
				11.75
			],
			"faces": {
				"north": {
					"uv": [
						3,
						10,
						4,
						16
					],
					"texture": "#5"
				},
				"east": {
					"uv": [
						2,
						10,
						3,
						16
					],
					"texture": "#5"
				},
				"south": {
					"uv": [
						1,
						10,
						2,
						16
					],
					"texture": "#5"
				},
				"west": {
					"uv": [
						0,
						10,
						1,
						16
					],
					"texture": "#5"
				},
				"up": {
					"uv": [
						4,
						10,
						3,
						8
					],
					"texture": "#5"
				},
				"down": {
					"uv": [
						3,
						8,
						2,
						10
					],
					"texture": "#5"
				}
			}
		},
		{
			"from": [
				3.8499999999999996,
				7.45,
				6.25
			],
			"to": [
				9.35,
				20.95,
				11.75
			],
			"faces": {
				"north": {
					"uv": [
						3,
						10,
						4,
						16
					],
					"texture": "#5"
				},
				"east": {
					"uv": [
						0,
						10,
						1,
						16
					],
					"texture": "#5"
				},
				"south": {
					"uv": [
						1,
						10,
						2,
						16
					],
					"texture": "#5"
				},
				"west": {
					"uv": [
						2,
						10,
						3,
						16
					],
					"texture": "#5"
				},
				"up": {
					"uv": [
						4,
						10,
						3,
						8
					],
					"texture": "#5"
				},
				"down": {
					"uv": [
						3,
						8,
						2,
						10
					],
					"texture": "#5"
				}
			}
		}
	],
	"display": {
		"head": {
			"rotation": [
				0,
				180,
				0
			]
		}
	}
}