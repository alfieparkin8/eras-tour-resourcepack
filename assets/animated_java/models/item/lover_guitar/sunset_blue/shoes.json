{
	"parent": "animated_java:item/lover_guitar/shoes",
	"textures": {
		"2": "animated_java:item/lover_guitar/lover_blue"
	}
}