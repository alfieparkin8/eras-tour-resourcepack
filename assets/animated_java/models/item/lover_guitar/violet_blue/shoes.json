{"parent":"animated_java:item/lover_guitar/shoes","textures":{"1":"minecraft:block/guitars/lover_blue","2":"minecraft:block/outfits/lover/violet_layer_1"}}