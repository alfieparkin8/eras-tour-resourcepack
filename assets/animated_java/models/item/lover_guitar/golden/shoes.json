{"parent":"animated_java:item/lover_guitar/shoes","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1","3":"minecraft:block/outfits/lover/golden_boots_layer_1"}}